demos/demo-react-ts/src/components/screens/IncomingScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect } from "react";
+import {
+  CallButton,
+  ContactLinkButton,
+  ContactLinkButtonContent,
+  EndCallButton,
+  IncomingCallContactDisplay,
+  IncomingCallNumber,
+  IncomingCallText,
+  IncomingScreenWrapper,
+  Row,
+  Timer,
+  Wrapper,
+} from "../Components";
+import { ScreenProps } from "../../types/ScreenTypes";
+import {
+  EndCallSvg,
+  ExternalLinkSvg,
+  SprocketSvg,
+  StartCallSvg,
+} from "../Icons";
+
+function IncomingScreen({
+  handleNextScreen,
+  incomingNumber,
+  handleEndCall,
+  cti,
+  startTimer,
+  callStatus,
+  incomingContactName,
+}: ScreenProps) {
+  const onAnswerCall = () => {
+    const callStartTime = Date.now();
+    cti.callAnswered();
+    startTimer(callStartTime);
+    handleNextScreen();
+  };
+
+  const onEndCall = () => {
+    cti.callEnded({
+      callEndStatus: callStatus.INTERNAL_COMPLETED,
+    });
+    handleEndCall();
+  };
+
+  return (
+    <IncomingScreenWrapper>
+      <div style={{ textAlign: "center" }}>
+        <IncomingCallText>Incoming call...</IncomingCallText>
+        {incomingContactName ? (
+          <>
+            <IncomingCallContactDisplay>
+              {incomingContactName}
+            </IncomingCallContactDisplay>
+            <IncomingCallNumber>{incomingNumber}</IncomingCallNumber>
+          </>
+        ) : (
+          <IncomingCallContactDisplay>
+            {incomingNumber}
+          </IncomingCallContactDisplay>
+        )}
```

Comment:
Couple of comments:
1. Could we format the incoming call number as `+1(415)787-8793`?
2. Do we have css to handle longer contact/company names since this changes the iframe dimensions?
<img width="393" alt="image" src="https://github.com/HubSpot/calling-extensions-sdk/assets/23175119/48deb85f-4f47-4cdb-8623-c555f8c0344b">